enterprise/static/enterprise/grant_data_sharing_permissions.js

```diff
@@ -1,17 +1,43 @@
+function togglePolicy (force_open) {
+    var policy = document.getElementsByClassName("consent-policy")[0];
+    currentDisplay = policy.style.display;
+    var icon = document.getElementById("consent-policy-dropdown-icon");
+    if ((currentDisplay === "none") || force_open) {
+        policy.style.display = "block";
+        icon.className = "fa fa-chevron-down fa-stack-1x";
+        location.href = "#";
+        location.href = "#consent-policy-dropdown-bar";
```

Comment:
I found some comments online about a bug that causes location.href to fragments in some WebKit browsers to not work unless an empty fragment is passed first.